sql/src/main/java/io/crate/analyze/repositories/RepositorySettingsModule.java

```diff
@@ -138,4 +97,28 @@ protected void configure() {
         typeSettingsBinder.addBinding(HDFS).toInstance(HDFS_SETTINGS);
         typeSettingsBinder.addBinding(S3).toInstance(S3_SETTINGS);
     }
+
+    private static Map<String, Setting> convertSettingListToMap(List<Setting> settingList) {
```

Comment:
```suggestion
    private static Map<String, Setting> groupSettingsByKey(List<Setting> settings) {
```

Otherwise you'll have to read the implementation to know what it does exactly - making this abstraction more complex than useful